docs/index.asciidoc

```diff
@@ -55,6 +66,19 @@ input plugins.
 How many times to iterate.
 This is typically used only for testing purposes.
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: `clock` counter field added at root level
+** `v1`: ECS compliant `[event][sequence]` counter field added to the event
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`.
+
+Refer to <<plugins-{type}s-{plugin}-ecs,ECS mapping>> for detailed information.
```

Comment:
```suggestion
Controls this plugin's compatibility with the {ecs-ref}[Elastic Common Schema (ECS)]. 
Refer to <<plugins-{type}s-{plugin}-ecs>> in this topic for detailed information.
```

Reply:
The link probably shouldn't say "ECS mapping" since the ECS topic in this plugin doesn't have that title.